AmericanMedicalAssociation/ama-style-guide-2
# EWL-8714: trending style-guide page

## Ticket(s)
**Do not** submit a Pull Request without a relevant JIRA ticket or Github issue! If you are creating a new pattern or feature, create an issue describing the need for this feature in Github **first** and then link to it below.

**Github Issue**
- N/A

**Jira Ticket**
- [EWL-8714: Trending index page in Styleguide](https://issues.ama-assn.org/browse/EWL-8714)

## Description
Adds various templates to display new Trending page in Style Guide.


## To Test
- Link styleguides
- run gulp serve to open styleguide
- Under Pages tab verify "Trending" appears toward bottom of list
- Select Trending and verify page matches styling in the following [Zeplin](https://app.zeplin.io/project/59dd1f9c36a8b21d569bb9e8/screen/60198c198a4c7b01b5bc37b5)

## Visual Regressions

- N/A

## Relevant Screenshots/GIFs
- N/A


## Remaining Tasks
- N/A


## Additional Notes
- One note, the logo within the membership block is not displaying, this seems to be happening across all style-guide pages. Will request a ticket for this specific fix.

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
